transports/janus_rabbitmq.c

```diff
@@ -684,6 +521,201 @@ int janus_rabbitmq_init(janus_transport_callbacks *callback, const char *config_
 	return -1;
 }
 
+int janus_rabbitmq_connect(void) {
+	rmq_client->connected = 0;
+	/* Connect */
+	rmq_client->rmq_conn = amqp_new_connection();
+	amqp_socket_t *socket = NULL;
+	amqp_queue_declare_ok_t *declare = NULL;
+	int status;
+	JANUS_LOG(LOG_VERB, "Creating RabbitMQ socket...\n");
+	if (ssl_enabled) {
```

Comment:
Code style: no space between `if` and the bracket (that was indeed there already in the existing code, but I may have missed it at the time, so we may as well fix it :grin: )